src/constants.py

```diff
@@ -0,0 +1,47 @@
+"""Project Global Constants. """
+import json
+import os
+
+from dotenv import load_dotenv
+from duneapi.types import Address
+from web3 import Web3
+
+load_dotenv()
+
+COW_TOKEN_ADDRESS = Address("0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB")
+# Things requiring network
+# TODO - this is such a mess!
+network_string = os.environ.get("NETWORK", "mainnet")
```

Comment:
I like that you write some strings in upper case in order to indicate that you will use them as constants. Why don't you do it everywhere?

Reply:
fixed..